dist/flags.py

```diff
@@ -113,6 +113,7 @@
         'SESSION_INTERNAL',
         'SESSION_LOCKED_CHECKPOINT',
         'SESSION_LOCKED_HANDLE_LIST',
+        'SESSION_LOCKED_METADATA',
```

Comment:
Note that the metadata lock is closely related to the turtle lock -- I think they can be combined.